src/styles/blocks/header.scss

```diff
@@ -38,5 +38,12 @@
     text-transform: uppercase;
     font-size: 12px;
     font-weight: 500;
+
+    transition-duration: $trans-dur;
+    transition-timing-function: $smooth;
+
```

Comment:
let's combine everything transition-related in this block into a single property

Reply:
fixed